src/cacheServerClient/cacheServerClient.ts

```diff
@@ -10,15 +10,23 @@ export class CacheServerClient implements ICacheServerClient {
   private httpClient: AxiosInstance;
   private isAlreadyFetchingAccessToken = false;
   private failedRequests: Array<() => void> = [];
+  private authEnabled: boolean;
 
-  constructor({ url }: { url: string }) {
+  constructor({
+    url,
+    cacheServerSupportsAuth = true
+  }: {
+    url: string;
+    cacheServerSupportsAuth?: boolean;
```

Comment:
@dwojno How does the consumer (e.g. Switchboard app) of iam-client-lib set this `cacheServerSupportsAuth` param?

Reply:
Like this:
```
        const cacheClient = new CacheServerClient({ url: 'http://cache-server.com', cacheServerSupportsAuth: false })

        this._iam = new IAM({
          rpcUrl: 'https://volta-rpc.energyweb.org/',
          chainId: 73799,
          cacheClient // optional
        });
```

Reply:
Ah right, I forgot they pass in the cache server client.